app/assets/javascripts/search.js

```diff
@@ -23,4 +23,25 @@ jQuery(document).ready(function() {
     }
   })
 
+  $('#questions_searchform').submit(function(e){ 
+    e.preventDefault()
+    window.location = '/search/questions/'+$('#questions_searchform_input').val()
```

Comment:
@ujithaperera - please note that @ananyo2012 is beginning to use the search API, and if you can both coordinate as you move forward, that would be great, thanks!